index.bs

```diff
@@ -2525,7 +2525,7 @@ partial interface MLGraphBuilder {
 </div>
 
 <div class="note">
-Although operations *greaterOrEqual* and *lesserOrEqual* can each be implemented in terms of operations *not*, *lesser*, and *greater* in other words `greater-or-equal(a, b)` is `not(lesser(a, b))`, they are specifically defined to handle NaN cases and for performance reason to avoid double comparisons.
+Although operations {{MLGraphBuilder/greaterOrEqual()}} and {{MLGraphBuilder/lesserOrEqual()}} can each be implemented in terms of operations {{MLGraphBuilder/not()}}, {{MLGraphBuilder/lesser()}}, and {{MLGraphBuilder/greater()}} in other words `greaterOrEqual(a, b)` is `not(lesser(a, b))`, they are specifically defined to handle NaN cases and for performance reason to avoid double comparisons.
```

Comment:
The spec usually uses `builder` in the emulation sample. Should we also be consistent in this compact emulation code? like `greaterOrEqual(a, b)` can be implemented by `builder.not(builder.lesser(a, b))`?

Reply:
Makes sense - done!